src/EditorFeatures/CSharp/EmbeddedLanguages/CSharpEmbeddedLanguageEditorFeaturesProvider.cs

```diff
@@ -1,20 +1,22 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System;
 using System.Composition;
 using Microsoft.CodeAnalysis.CSharp.EmbeddedLanguages.LanguageServices;
 using Microsoft.CodeAnalysis.CSharp.Features.EmbeddedLanguages;
 using Microsoft.CodeAnalysis.Editor.EmbeddedLanguages;
-using Microsoft.CodeAnalysis.Features.EmbeddedLanguages;
+using Microsoft.CodeAnalysis.EmbeddedLanguages.LanguageServices;
 using Microsoft.CodeAnalysis.Host.Mef;
 
 namespace Microsoft.CodeAnalysis.CSharp.Editor.EmbeddedLanguages
 {
-    [ExportLanguageService(typeof(IEmbeddedLanguageEditorFeaturesProvider), LanguageNames.CSharp), Shared]
+    [ExportLanguageService(typeof(IEmbeddedLanguagesProvider), LanguageNames.CSharp, ServiceLayer.Editor), Shared]
```

Comment:
IEmbededLanguagesProvider seems completely new pattern than anything else we have? I guess people were okay introducing this new concept?

Reply:
@heejaechang Can you clarify?  This is a pretty normal language service, similar to how we've plugged in a lot of language specific behaviors.

Reply:
> IEmbededLanguagesProvider seems completely new pattern than anything else we have? I guess people were okay introducing this new concept?

This was part of a different review. The issue addressed here is the separate export of three related interfaces, where the code should have uniformly exported the base interface.